jbpm-process-svg/src/main/java/org/jbpm/process/svg/processor/StunnerSVGProcessor.java

```diff
@@ -87,8 +93,12 @@ private void processNode(final Node parent, final String nodeId) {
                         Element border = Objects.equals(parameters.get("shapeType"), "BORDER") ? (Element) node : nodeSummary.getBorder();
                         Element background = Objects.equals(parameters.get("shapeType"), "BACKGROUND") ? (Element) node : nodeSummary.getBackground();
                         RenderType renderType = RenderType.valueOf(Optional.ofNullable(parameters.get("renderType")).orElse(nodeSummary.getRenderType().orElse(RenderType.STROKE).name()));
+                        Element plusButton = null;
+                        if (subProcessLinks !=null && subProcessLinks.containsKey(nodeId) && value.equals(nodeId + "undefined")) {
```

Comment:
Hey  @bxf12315 

I just created the PR: https://github.com/kiegroup/kie-wb-common/pull/3633

So now you can use the ids from the "source" SVG, instead of this tricky "undefined" :)

Hope it works, I've tested it locally and works fine!

Reply:
it's works fine in my PR.